Update roles and add guidelines

Delphine opened the pull request.

flodolo commented on src/community/l10n_community_roles.md:
```diff
@@ -20,70 +30,156 @@ L10n-driver responsibilities and expectations are multifaceted, being that in ad
+
+If you have contacted members of your team and received no responses, have questions about these guidelines or want to suggest improvements, don’t hesitate to get in touch with the localization staff:
+
+1. You can use [email](pontoon-team@mozilla.com) and [Matrix](https://matrix.to/#/#l10n-community:mozilla.org) to reach out to the entire team.
```
```suggestion
1. You can use [email](mailto:pontoon-team@mozilla.com) and [Matrix](https://matrix.to/#/#l10n-community:mozilla.org) to reach out to the entire team.
```

flodolo commented on src/community/l10n_community_roles.md:
```diff
@@ -20,70 +30,156 @@ L10n-driver responsibilities and expectations are multifaceted, being that in ad
+If you have contacted members of your team and received no responses, have questions about these guidelines or want to suggest improvements, don’t hesitate to get in touch with the localization staff:
+
+1. You can use [email](pontoon-team@mozilla.com) and [Matrix](https://matrix.to/#/#l10n-community:mozilla.org) to reach out to the entire team.
+2. You can also find the member of staff in charge of each project [directly in Pontoon](https://www.google.com/url?q=https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/teams_projects.html%23project-page&sa=D&source=editors&ust=1727922613572178&usg=AOvVaw1OILveFC-KMISo0no5zV5l).
```
Wrong link (also use a relative one).

flodolo commented on src/community/l10n_community_roles.md:
```diff
@@ -20,70 +30,156 @@ L10n-driver responsibilities and expectations are multifaceted, being that in ad
+
+### I am a Translator and want to become a Manager, what do I do?
+
+Try to get in touch with the current Managers and Translators from your locale, to surface your request (you can find them under the Contributor view in your [team page](https://www.google.com/url?q=https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/teams_projects.html%23teams-page&sa=D&source=editors&ust=1727922613572868&usg=AOvVaw1yvThhddMWNZwZX-tOYzjn) on Pontoon). Check also ...
```
Wrong link.

Delphine commented on src/community/l10n_community_roles.md:
```diff
@@ -1,14 +1,24 @@
+
+In the following sections, we’ll review the current roles and expectations and suggest guidelines for when role criteria are not met — and how the localization staff can get involved and help in these different scenarios.
+
+## Roles: responsibilities and expectations
```
I thought that we'd just keep the l10n-drivers section as is, since there wasn't any discussion about changing it or getting feedback on that section.
I do understand we want to stay away now from the term l10-drivers and I can update that - but did we want to get rid of that section entirely?

Delphine replied:
Got rid of that section as per our 1:1 discussion

flodolo commented on src/community/l10n_community_roles.md:
```diff
@@ -1,89 +1,150 @@
+
+If a localizer has attempted to reach out to Managers or Translators but has not received a satisfactory response (or any response) within 2 weeks, it’s advisable to contact the localization staff for assistance.
+
+## Common issues & quick references
```
```suggestion
## Common issues and references
```

flodolo commented on src/community/l10n_community_roles.md:
```diff
@@ -1,89 +1,150 @@
-# Roles within Mozilla l10n communities
+# Introduction to localization roles and community guidelines
```
```suggestion
# Roles and guidelines in the localization community
```

Why "Introduction"? This document is supposed to be the reference.

Delphine replied:
This was more of a title for the first part of the doc.

flodolo reviewed (changes requested):
Please fix SUMMARY.md in this PR.

Delphine commented:
@bcolsson Would you mind taking a last look at this PR, since flod is now on PTO?
I addressed all comments, and added the update to SUMMARY.md in this PR - so I'm fairly confident this is good to go. But please let me know :) thanks!
